airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/manifest_reference_resolver.py

```diff
@@ -75,40 +77,38 @@ class ManifestReferenceResolver:
     limit-ref: 50
     ```
 
-    whereas here we want to access the `nested.path` value.
+    whereas here we want to access the `nested/path` value.
     ```
     nested:
         path: "first one"
-    nested.path: "uh oh"
-    value: "ref(nested.path)
+    nested/path: "uh oh"
+    value: "#/nested/path
     ```
     will produce the following definition:
     ```
     nested:
         path: "first one"
-    nested.path: "uh oh"
+    nested/path: "uh oh"
     value: "uh oh"
     ```
 
     to resolve the ambiguity, we try looking for the reference key at the top level, and then traverse the structs downward
     until we find a key with the given path, or until there is nothing to traverse.
     """
 
-    ref_tag = "$ref"
-
     def preprocess_manifest(self, manifest: Mapping[str, Any]) -> Mapping[str, Any]:
         """
         :param manifest: incoming manifest that could have references to previously defined components
         :return:
         """
         return self._evaluate_node(manifest, manifest, set())
 
-    def _evaluate_node(self, node: Any, manifest: Mapping[str, Any], visited: Set):
+    def _evaluate_node(self, node: Any, manifest: Mapping[str, Any], visited: Set) -> Any:
         if isinstance(node, dict):
             evaluated_dict = {k: self._evaluate_node(v, manifest, visited) for k, v in node.items() if not self._is_ref_key(k)}
-            if self.ref_tag in node:
+            if REF_TAG in node:
```

Comment:
just a q: why are we keeping backwards compatibility?

Reply:
We are no longer supporting the `*ref` format, so aren't keeping backwards compatibility there. The `$ref` format is supported by JSON Schema so we're still using it to identify refs whose value should be splatted into the referencer.

Reply:
ah got it. Thanks for the 🥁 pointer 🥁